bootstrap-orbit.ps1

```diff
@@ -65,7 +65,7 @@ You can call 'pip3 show -f conan' to figure out where conan.exe was placed.
 # Start build
 if ($args) {
   & "$PSScriptRoot\build.ps1" $args
-} elseif (conan remote list | Select-String -NotMatch "Disabled:" | Select-String "artifactory:") {
+} elseif (& $conan.Path remote list | Select-String -NotMatch "Disabled:" | Select-String "artifactory:") {
   & "$PSScriptRoot\build.ps1" "default_relwithdebinfo" "ggp_relwithdebinfo"
 } else {
   & "$PSScriptRoot\build.ps1"
```

Comment:
nit: It seems like build.sh/ps1 also does some check whether arguments are provided or not. Maybe these lines can go into build.sh/ps1. I think that would simplify this a tiny little bit

Reply:
Unfortunately this won't work. When the build script is called without arguments it will only build the default profile (`default_relwithdebinfo`). And when introducing this logic I didn't want to change that. Everybody is used to that behaviour.

So I have to decide here whether I wanna build the profiles provided, or default and ggp if possible, or fallback to the build-script default which means only the default profile.

Does that make sense?

Reply:
Yes, it does